bee-api/bee-rest-api/src/endpoints/config.rs

```diff
@@ -217,3 +220,14 @@ impl RestApiConfig {
         self.white_flag_solidification_timeout
     }
 }
+
+pub fn route_to_regex(route: &str) -> String {
```

Comment:
this is `pub`, but doesn't require documentation? Can this be `pub(crate)` then?